experiments.json

```diff
@@ -1,15 +1,46 @@
 {
-  "onboarding-a": {
-    "match": {},
+  "search-term": {
+    "match": {
+      "version": "^4[6-9]\\..*"
+    },
     "buckets": {
       "min": "0",
       "max": "50"
     }
```

Comment:
We just want to roll out the search-term change to 50% of users? This could be interesting to try and monitor what happens.

We could add an experiment extra to the AWESOMESCREEN session to see if people log different events depending if they're in the experiment or not.